DEPLOY.md

```diff
@@ -85,7 +89,8 @@ This file needs some customisation, depending on your setup.
 1. Change region, s3Endpoint and iamEndpoint to match your HyperStore setup
 1. For greenfield: delete bucketName and optionally set the storage policy to use for new buckets. Omit the storagePolicyId line to use the default policy. To find the policy ID, navigate to the Cluster->Storage Policies page on the CMC, select View/Edit for the policy, and copy the ID field (above the Policy Name field)
 1. For brownfield: specify an already created bucket name and delete reclaimPolicy and storagePolicyId
-1. Optionally specify an iam policy document to override default read+write access to the bucket. You do not need to specify `Resource` fields - they will be set to only allow access to the claimed bucket. For example, to grant read-only access to the bucket:
+1. Optionally disable IAM user creation for bucket access by setting `createBucketUser` to `"no"`. A separate secret can be used to hold a shared credentials granted to all object bucket claims for this storage class by setting `bucketClaimUserSecretName` and `bucketClaimUserSecretNamespace`. If this secret is unset, the credentials from the `secretNamespace/secretName` are provided to object bucket claims. 
+1. Optionally specify an IAM policy document to override default read+write access to the bucket. You do not need to specify `Resource` fields - they will be set to only allow access to the claimed bucket. For example, to grant read-only access to the bucket:
```

Comment:
Move up 1 line?  [So the ordering in the list matches the ordering in the example yaml.]

Reply:
I'll change the yaml - moving the IAM policy item up one line didn't work nicely due to the sample yaml in that item.